include/grpc/grpc.h

```diff
@@ -387,7 +386,8 @@ grpc_call *grpc_channel_create_registered_call(
    completion of type 'tag' to the completion queue bound to the call.
    The order of ops specified in the batch has no significance.
    Only one operation of each type can be active at once in any given
-   batch. */
+   batch. 
+   THREAD SAFETY: grpc_call_start_batch is thread-compatible */
```

Comment:
It's totally fine to be doing so. It's actually completely threadsafe to
start each different op type from a separate thread currently, but that
won't always be the case.

On Mon, Jun 22, 2015 at 9:57 AM Jan Tattermusch <notifications@github.com>
wrote:

> In include/grpc/grpc.h
> <https://github.com/grpc/grpc/pull/2047#discussion_r32956788>:
>
> > @@ -387,7 +386,8 @@ grpc_call *grpc_channel_create_registered_call(
> >     completion of type 'tag' to the completion queue bound to the call.
> >     The order of ops specified in the batch has no significance.
> >     Only one operation of each type can be active at once in any given
> > -   batch. */
> > +   batch.
> > +   THREAD SAFETY: grpc_call_start_batch is thread-compatible */
>
> Do you mean I can only be sending from one thread and receiving from one
> (possibly different) thread? That would be very broken (at least for C#).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2047/files#r32956788>.
>